docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,32 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking the DeepCell Kiosk
+---------------------------------------------------------
+
+The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To reproduce the cost and timing benchmarks reported in the DeepCell Kiosk paper, please refer to our `figure creation repository <https://github.com/vanvalenlab/kiosk-console/pull/github.com/vanvalenlab/publication-figures>`_. To run your own benchmarking, please read below.
+
+1. If you don't already have a cloud storage bucket for use with the DeepCell Kiosk, you should create one now. It's fine to reuse this bucket across multiple DeepCell Kiosk clusters.
+
+2. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
+
+    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ for an example.
```

Comment:
This URL doesn't provide access. I think there's another URL structure we can use that does allow access to unauthenticated accounts? The bucket has public read.

Reply:
Nevermind, it was a bucket configuration issue. I made the bucket public read for all users and the link works.

Reply:
Yeah, good catch. It should be `http://storage.googleapis.com/kiosk-benchmarking`. That just returns a raw XML file, though. You think we need to provide some guidance on how to navigate that file?

Reply:
I'm getting confused by where comments go, but I mentioned elsewhere that the issue was actually the access rights to the bucket. It's been resolved on the bucket side and the original link should work as intended.